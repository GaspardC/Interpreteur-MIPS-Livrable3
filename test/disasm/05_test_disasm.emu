#-------
#TEST_RETURN_CODE=PASS           
#TEST_COMMENT="Teste fonctionnement standard"
#-------

load boucle.o
disasm 0x3000:3010
