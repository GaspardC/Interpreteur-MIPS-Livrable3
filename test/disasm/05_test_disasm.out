3000 :: 2009000c	ADDI $t1, $zero, 12
3004 :: 00005020	ADD $t2, $zero, $zero
3008 :: 214a0001	ADDI $t2, $t2, 1
300c :: 1549fffe	BNE $t2, $t1, -8
3010 :: 00000000	NOP
